app/components/provider_interface/header_component.html.erb

```diff
@@ -1,4 +1,4 @@
-<header class="govuk-header <%= classes %>" role="banner" data-module="govuk-header">
+<header class="govuk-header provider-header <%= classes %>" role="banner" data-module="govuk-header">
   <div class="govuk-header__container govuk-width-container">
```

Comment:
@duncanjbrown I think there might be an easier way to get the desired result, that requires no changes to any CSS, but only to the HTML of the header. Instead of this:

```
<div class="govuk-header__container govuk-width-container">
  …
</div>
```

we can do this:

```
<div class="govuk-header__container">
  <div class="govuk-width-container">
    …
  </div>
</div>
```

Reply:
Much better! Updated